src/main/java/com/flowable/flowableboot/model/UmsTask.java

```diff
@@ -0,0 +1,190 @@
+package com.flowable.flowableboot.model;
+
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Objects;
+
+@Entity
+public class UmsTask {
```

Comment:
Add createdDate and updatedDate as metadata here - they likely won't show in the UI, but all database items should have these as a method of auditing.  My instinct is that receivedDate will be editable by the user, but createdDate will need to track the creation of the data item - i.e., a user receives a task on Monday, but is only able to add the task to UMS on Thursday.

Usually, things like Id, createdDate, and updatedDate are in a 'mapped super class' that extends your database items in general.  See https://vladmihalcea.com/how-to-inherit-properties-from-a-base-class-entity-using-mappedsuperclass-with-jpa-and-hibernate/

Reply:
A base entity was added. I did not know if we should include a version and what convention was typically used for that.

Reply:
I don't think a 'version' is necessary, just an updated date.